website/docs/r/vapp_vm.html.markdown

```diff
@@ -124,6 +124,7 @@ The following arguments are supported:
 * `vdc` - (Optional; *v2.0+*) The name of VDC to use, optional if defined at provider level
 * `vapp_name` - (Required) The vApp this VM should belong to.
 * `name` - (Required) A unique name for the VM
+* `computer_name` - (Optional; *v2.5+*) A unique name 
```

Comment:
For clarity, I'd like to see the long description of this field, copied either from API docs or UI.

Reply:
Copied from API description.